data_steward/resources/fields/operational_pii_fields.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "type": "string",
+    "name": "observation_source_value",
+    "mode": "nullable"
```

Comment:
Do we really want or expect these fields to be nullable?  It's easier to relax a `mode` than it is to strengthen one.  

Also, if you wouldn't mind adding descriptions that would be really helpful for future maintenance efforts.  Out of context, I'm not sure what `total_count` represents.

Reply:
These were the tables given to us, even I am not sure about the total_count too. we are not using it in the query. I can make fields required.

Reply:
Interesting.  Who gave us the tables?